resources/serialization-order.edn

```diff
@@ -0,0 +1,137 @@
+{:Card       [:description
```

Comment:
If we're going to have a bespoke order defined in this file, then I think this (which seems to be pretty straight from DB column order?) could be improved a lot. There are human-friendly fields like `Card.name` below potentially-huge embedded records like `Card.result_metadata`.

Since many of these files end up being hundreds of lines or more, we should take this opportunity to make the files friendly to human readers looking at the top "paragraph" of the files.

I'd also remark that some of the appdb columns don't get exported in YAML, and other columns are added during extraction that don't exist in the appdb.

Three things to do here:

1. Check this file against the `extract`ed forms or eyeball the `extract-one` implementations; some of these columns are dropped and some others are added.
2. Reorder this file to make the order human friendly.
3. Include several models that are missing here! DashboardCard, DashboardTab, Timeline and TimelineEvent, and more.
    - Remember that many models are embedded in others; we still want a consistent order for DashboardCard's keys even though they're embedded on Dashboards, etc.

## Proposed order

Roughly the order should be:
- meaningful human things (`:name`, `:description`, `:archived`, `Database.engine`, `Card.dataset`)
- `:serdes/meta`
- short and fairly readable internals (`:database_id`, `:table_id`, `:collection_position`, `:creator_id`, `Card.display`, `Database.details`, etc.)
- short(ish) and less readable internals (`embedding_params`, `dataset_query`, etc.)
- huge embedded lists (`dashboard_cards`, `result_metadata`, `parameters`)

Reply:
Just updated to that message

Reply:
LGTM now.